ci/terraform/oidc/authentication-callback.tf

```diff
@@ -48,6 +48,8 @@ module "authentication_callback" {
     REDIS_KEY                                   = local.redis_key
     SUPPORT_AUTH_ORCH_SPLIT                     = var.support_auth_orch_split
     TXMA_AUDIT_QUEUE_URL                        = module.oidc_txma_audit.queue_url
+    OIDC_API_BASE_URL                           = local.api_base_url
```

Comment:
is this needed then ?

Reply:
needed for other envs

Reply:
yep it's the base redirect URL in envs other than staging and prod